demos/list.html

```diff
@@ -60,32 +62,139 @@
-      </aside>
-      <div class="mdc-form-field">
-        <div class="mdc-checkbox">
-          <input type="checkbox"
-                 class="mdc-checkbox__native-control"
-                 id="toggle-rtl"
-                 aria-labelledby="toggle-rtl-label" />
-          <div class="mdc-checkbox__background">
-            <svg class="mdc-checkbox__checkmark" viewBox="0 0 24 24">
-              <path class="mdc-checkbox__checkmark__path" fill="none" stroke="white"
-                    d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
-            </svg>
-            <div class="mdc-checkbox__mixedmark"></div>
+      <div class="mdc-toolbar-fixed-adjust"></div>
+      <section class="hero">
+        <ul class="mdc-list mdc-list--two-line mdc-list--avatar-list two-line-avatar-text-icon-demo">
+          <li class="mdc-list-item">
+            <span class="mdc-list-item__start-detail grey-bg" role="presentation">
+              <i class="material-icons" aria-hidden="true">folder</i>
+            </span>
+            <span class="mdc-list-item__text">
+              Photos
+              <span class="mdc-list-item__text__secondary">Jan 9, 2014</span>
+            </span>
+            <a href="#" class="mdc-list-item__end-detail material-icons"
+               aria-label="View more information" title="More info"
+               onclick="event.preventDefault();">
+              info
+            </a>
+          </li>
+          <li class="mdc-list-item">
+            <span class="mdc-list-item__start-detail grey-bg" role="presentation">
+              <i class="material-icons" aria-hidden="true">folder</i>
+            </span>
+            <span class="mdc-list-item__text">
+              Recipes
+              <span class="mdc-list-item__text__secondary">Jan 17, 2014</span>
+            </span>
+            <a href="#" class="mdc-list-item__end-detail material-icons"
+               aria-label="View more information" title="More info"
+               onclick="event.preventDefault();">
+              info
+            </a>
+          </li>
+          <li class="mdc-list-item">
+            <span class="mdc-list-item__start-detail grey-bg" role="presentation">
+              <i class="material-icons" aria-hidden="true">folder</i>
+            </span>
+            <span class="mdc-list-item__text">
+              Work
+              <span class="mdc-list-item__text__secondary">Jan 28, 2014</span>
+            </span>
+            <a href="#" class="mdc-list-item__end-detail material-icons"
+               aria-label="View more information" title="More info"
+               onclick="event.preventDefault();">
+              info
+            </a>
+          </li>
+        </ul>
+      </section>
+
+      <section class="preamble mdc-typography--body1">
+        <aside>
+          <p>
+          <em>NOTE:</em> For the purposes of this demo, we've set a max-width of 600px on all
+          <code>mdc-list</code> elements surrounded by a 1px border. This is
```

Comment:
nit: can we change "surrounded by a 1px border" to ", and surrounded them by a 1px border". Otherwise, it reads as if we've _only_ set a max-width of 600px on bordered lists, when in reality we set it on all lists.